config/application.yml.default

```diff
@@ -91,7 +91,7 @@ doc_auth_error_dpi_threshold: 290
 doc_auth_error_glare_threshold: 40
 doc_auth_error_sharpness_threshold: 40
 doc_auth_max_attempts: 5
-doc_auth_max_capture_attempts_before_tips: 3
+doc_auth_max_capture_attempts_before_tips: 20 
```

Comment:
is this number allowed to exceed doc_auth_max_attempts?

Reply:
As far as I know, yes.